deploy/cloud/README.md

```diff
@@ -1,40 +1,69 @@
 # Cloud Deployment
 
 **Motivation**
-Provide an easy and deterministic way to setup latest cloud environment so it can be monitored and used properly.
+Provide an easy and deterministic way to set up latest cloud environment, so it can be monitored and used properly.
+
+This guide deploys both an Elastic cloud environment, and an AWS EKS cluster. To only deploy specific resources, check out the examples section.
 
 **Prerequisite**
-* [terraform](https://www.terraform.io/)
+* [Terraform](https://developer.hashicorp.com/terraform/downloads)
+* the AWS CLI, [installed](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) and [configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
+* [AWS IAM Authenticator](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html)
+* the [Kubernetes CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl/), also known as `kubectl`
```

Comment:
can be removed and changed to activate hermit env instead

Reply:
I think having both hermit and the explicit documentation is nicer

Reply:
can lead to confusion if hermit env is not installed and instead we will install those dependencies manually (different versions than the pinned ones) this already happened with our main README, so we replaced it with how to activate hermit.

Reply:
done in https://github.com/elastic/cloudbeat/pull/582